### PR TITLE
Javalab: add button to enable peer review

### PR DESCRIPTION
**Pull request by bencodeorg**
Add placeholder button for enabling peer review in the "Review" tab in Javalab.

image

Admittedly a little more styling that needs to be done to get the checkbox and label to line up, but figure we can clean that up as we build the Review tab out a bit more.

Figma mock here:

image

## Testing story

Tested manually -- confirmed state changes worked as expected. Also that horizontal resize looks ok for now (on my laptop at least).

### CODE REVIEW

**Comment by bencodeorg** [comment]
Merged staging to resolve unrelated failing test, should be good to review.

File: apps/src/templates/instructions/ReviewTab.jsx [hunk 1]
```diff
@@ -1,9 +1,41 @@
...
- */
-export default function ReviewTab() {
-  return <p>Review...</p>;
+import React, {Component} from 'react';
+import javalabMsg from '@cdo/javalab/locale';
+
+export default class ReviewTab extends Component {
+  state = {
+    readyForReview: false
+  };
```
Comment (sanchitmalhotra126) [inline apps/src/templates/instructions/ReviewTab.jsx:7]: Will this be driven by Redux? Or I guess that's something we can revisit when more of the feature is in?

**Review by sanchitmalhotra126 — APPROVED** [review]
LGTM!

File: apps/src/templates/instructions/ReviewTab.jsx (hunk 1 above)
Comment (bencodeorg) [reply apps/src/templates/instructions/ReviewTab.jsx:7]: mmm I hadn't really considered yet...I'm not sure to what degree whether a project is ready to review will be used by other parts of Javalab (ie, whether we'd need to keep it in Redux). I think we can wait and see until we build this out a little more.

**Review by molly-moen — APPROVED** [review]

**Pull request merged**